config/org.eclipse.jdt.core.prefs

```diff
@@ -58,7 +58,7 @@ org.eclipse.jdt.core.compiler.problem.typeParameterHiding=error
 org.eclipse.jdt.core.compiler.problem.fieldHiding=error
 #We can not efnforce this rule as we are library and we keep deprecated stuff for some time
 org.eclipse.jdt.core.compiler.problem.deprecation=ignore
-#We can not use try-with-resources till we use cobertura for coverage
+# Too many false positives. LocalizedMessage$Utf8Control.newBundle for example.
```

Comment:
Build log https://gist.github.com/pbludov/1a3683deb52a1ba97cb268e026f224a8
Idea inspection TryFinallyCanBeTryWithResources is more smart.

Reply:
> 3. ERROR in /home/travis/build/checkstyle/checkstyle/src/main/java/com/puppycrawl/tools/checkstyle/api/LocalizedMessage.java (at line 525)

Why isn't eclipse right?
Shouldn't this be the try-with-resource ?
````
if (stream != null) {
    try (Reader streamReader = new InputStreamReader(stream, StandardCharsets.UTF_8.name())) {
````

Reply:
Yes, this one can be.

Reply:
@pbludov Don't forget to update the comment since you converted the one that the comment says can't be.